include/openPMD/IO/AbstractIOHandlerHelper.hpp

```diff
@@ -37,7 +37,7 @@ namespace openPMD
      */
     std::shared_ptr< AbstractIOHandler >
     createIOHandler(
-        std::string const& path,
+        std::string path,
```

Comment:
makes sense to do this as well, no?

Reply:
Yes, this then can forward the string to the constructor of the appropriate IOHandler (which also take their path-strings by value).